app/src/main/cpp/skyline/services/codec/results.h

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright © 2020 Skyline Team and Contributors (https://github.com/skyline-emu/)
```

Comment:
I agree

Reply:
I disagree, can we move to another license?